Assets/Src/Auth/AuthTokenSaver/IAuthTokenSaver.cs

```diff
@@ -0,0 +1,13 @@
+using Src.Auth.JwtManagement;
+
+namespace Src.Auth.AuthTokenSaver
+{
+    public interface IAuthTokenSaver
+    {
+        public void TryGetTokenStoreByAuthType(out AbstractJwtStore store, AuthType providerType);
+        public void SaveAuthTokens(AbstractJwtStore store, AuthType providerType);
+
+        public bool TryGetLastLoginInfo(out AuthType authType);
+        public void UpdateLastLoginInfo(AuthType authType);
```

Comment:
LoginInfo in the name of this method sounds like there is some entity called LoginInfo, but there is no such entity. It was a bit confusing. Maybe add some docstring or rename methods to TryGetLastAuthType/UpdateLastAuthType or something like that?